actionmailer/lib/action_mailer/test_helper.rb

```diff
@@ -123,7 +123,7 @@ def assert_enqueued_emails(number, &block)
     #       ContactMailer.with(email: 'user@example.com').welcome.deliver_later
     #     end
     #   end
-    def assert_enqueued_email_with(mailer, method, args: nil, queue: "mailers", &block)
+    def assert_enqueued_email_with(mailer, method, args: nil, queue: "default", &block)
```

Comment:
For simplicity purpose, I have not use default_queue_name, because then we need to support prefixes and other queue name computations. So for now it's simple enough.

But if you think then maybe ActionMailer::Base.default_queue_name as default will reduce duplication (a little) I could just use it